pkg/server/routes.go

```diff
@@ -28,6 +28,7 @@ func ApiRouter(router *mux.Router, notificationChannel chan common.Notification)
 	router.HandleFunc("/hosts", HostsHandler)
 	router.HandleFunc("/long", LongRunningTaskHandler(notificationChannel))
 	router.HandleFunc("/bootstrap-vm", BootstrapVMHandler(notificationChannel))
+	router.HandleFunc("/ignition-configs", CreateIgnitionConfigsHandler)
```

Comment:
I like this way of defining routes. It would be nice to establish a standardized way of defining routes and handlers e.g.:
```
router.HandleFunc("/hosts", GetHostsHandler).Methods("GET")
router.HandleFunc("/hosts/{id}", GetHostHandler).Methods("GET")
router.HandleFunc("/hosts", CreateHostHandler).Methods("POST")
router.HandleFunc("/hosts/{id}", UpdateHostHandler).Methods("PATCH", "PUT")
router.HandleFunc("/hosts/{id}", DeleteHostHandler).Methods("DELETE")
```

For singleton resources:
```
router.HandleFunc("/install-config", GetInstallConfig).Methods("GET")
router.HandleFunc("/install-config", CreateInstallConfig).Methods("POST")
router.HandleFunc("/install-config", UpdateInstallConfig).Methods("PATCH", "PUT")
router.HandleFunc("/install-config", DeleteInstallConfig).Methods("DELETE")
```

Handlers for each resource could be held in a separate module. We could even omit `...Handler` in the handler function name.